index.html

```diff
@@ -52,5 +60,173 @@ <h1 class="secondHeroIntroHeading">
+            </div>
+          </div>
+        </div>
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="listLanguages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="listLanguages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="listLanguages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="aboutMe">
+        <div class="primaryTextAboutMe">
+          <h3 class="primaryTextAboutMeHeading">About me</h3>
+        </div>
+        <div class="supportingTextAboutMe">
+          <p class="supportingTextAboutMePara">
+            I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="aboutMeBtn">
+          <button type="button" class="get-my-resume-btn click">
+            Get My Resume
+          </button>
+        </div>
+      </section>
+      <section class="languageMainContainer">
+        <div class="languageSubContainer">
+          <img
+            src="pngFiles/ballIcon.png"
+            alt="languages"
+            width="327px"
+            height="128px"
+          />
+          <li class="languageContainerText">Languages</li>
+          <ul class="listLanguages">
+            <li class="language">JavaScript</li>
+            <li class="language">Ruby</li>
```

Comment:
- [ ] [OPTIONAL] Please in close your `li` tag inside a parent container `ul` instead of `div` tag. The `<li> ` HTML element is used to represent an item in a list. It must be contained in a parent element: an ordered list ( `<ol>` ), an unordered list ( `<ul>` ), or a menu ( `<menu> `):